src/chapters/thymeleaf-views/template-conditionals.rst

```diff
@@ -0,0 +1,232 @@
+Conditionals in a Template
+===========================
+
+In addition to iteration, Thymeleaf can also add or remove content on a
+webpage based on certain conditions. Going back to the coffee example, we could
+generate the ordered list ONLY IF ``coffeeOptions`` contains data. If the
+ArrayList is empty, then there is no need to include the ``<ol>`` element.
+Instead, the template could include a ``<p>`` element with text stating that
+there are no options to select.
+
+Just like the ``for/each`` syntax differs between Java and Thymeleaf, we need
+to examine how to include *conditionals* in our templates. The logic remains
+the same, but the implementation requires practice.
+
+Try It!
+--------
+
+Code along with the following video to gain more practice with Thymeleaf
+iteration and conditionals:
+
+.. todo:: ``hello-spring`` video #3 goes here...
+
+The text on this page and the previous one provides details for some of the
+concepts presented in the clip. Note that these summaries are NOT intended as
+a replacement for the walkthrough. To get better at coding, you need to
+actually CODE instead of just reading about how to do it.
```

Comment:
Same as before. I would make this the last section on the page before the concept checks.

Reply:
Moved.